release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,93 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+
+    leaf netconf-transport-porotocol {
+      type enumeration {
+        enum SSH_v2 {
```

Comment:
All enum arguments must be capitalized (refer to all other OC models)

Reply:
enum arguments has been capitalized